### PR TITLE
Remove usageBased container from LTSR operator

### PR DESCRIPTION
**Pull request by Pejdzor**
Remove usageBased Container Feature from License Service

### CODE REVIEW

**Comment by kflorianski-ibm** [comment]
See https://github.com/IBM/ibm-licensing-operator/pull/916#pullrequestreview-2109800387

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: ibm-licensing-operator-app
+  operators.operatorframework.io.bundle.package.v1: ibm-licensing-operator
```
Comment (ljeda) [inline bundle/metadata/annotations.yaml:6]: why we introduce this change here?

**Review by ljeda — CHANGES_REQUESTED** [review]

File: bundle/metadata/annotations.yaml (hunk 1 above)
Comment (Pejdzor) [reply bundle/metadata/annotations.yaml]: We are not supposed. I created 4 Pull Requests, and missed this. Good catch, thanks a lot

**Review by kflorianski-ibm — APPROVED** [review]

**Review by ljeda — APPROVED** [review]

**Pull request merged**